druid/src/widget/split.rs

```diff
@@ -91,6 +93,11 @@ impl<T> Split<T> {
         self.draggable = draggable;
         self
     }
+    /// Set whether the splitter handle is a solid rectangle or two parallel lines.
```

Comment:
```suggestion
    /// Builder-style method to set whether the splitter handle is drawn as a solid rectangle.
    ///
    /// If this is `false` (the default), it will be drawn as two parallel lines.
```